example/react/src/App.js

```diff
@@ -9,37 +9,42 @@ const auth = new GoTrueClient({
 })
 
 function App() {
-  let [session, setSession] = useState(auth.session())
+  let [session, setSession] = useState()
   let [email, setEmail] = useState(localStorage.getItem('email') ?? '')
   let [phone, setPhone] = useState(localStorage.getItem('phone') ?? '')
   let [password, setPassword] = useState('')
   let [otp, setOtp] = useState('')
   let [rememberMe, setRememberMe] = useState(false)
 
```

Comment:
I think we'll need to add a `useEffect` here that calls `getSession()` and then sets the result